src/backend/distributed/operations/healthcheck.c

```diff
@@ -0,0 +1,193 @@
+/*-------------------------------------------------------------------------
+ *
+ * healthcheck.c
```

Comment:
It might be very useful to give the high-level overview of how the UDFs work.

Reply:
minor: It might be slightly easy to read `health_check.c`, feel free to decide yourself